modules/proc_deploy-quay-openshift-operator.adoc

```diff
@@ -746,6 +789,47 @@ spec:
     skipSetup: true
 ```
 
+== Specifying the {productname} route
+{productname} makes use of an OpenShift route to enable ingress. The hostname
+for this route is automatically generated as per the configuration of the
+OpenShift cluster. Alternatively, the hostname for this route can be explicitly
+specified using the hostname property under the externalAccess field as shown below:
+
+```
+apiVersion: redhatcop.redhat.io/v1alpha1
+kind: QuayEcosystem
+metadata:
+  name: example-quayecosystem
+spec:
+  quay:
+    imagePullSecretName: redhat-pull-secret
+    externalAccess:
+      hostname: example-quayecosystem-quay-quay-enterprise.apps.openshift.example.com
+```
+
+== Specifying a {productname} configuration route
+
+During the development process, you may want to test the
+provisioning and setup of your {productname} server. By default,
+the operator will use the internal service to communicate with
+the configuration pod. However, when running external to the cluster,
+you will need to specify the ingress location for which the setup process
```

Comment:
As `ingress` is an overloaded term in kubernetes, it might be helpful to change the term from `ingress` to `hostname` in this case.

Reply:
Done.